src/PIL/ImageDraw.py

```diff
@@ -364,15 +364,15 @@ def floodfill(image, xy, value, border=None, thresh=0):
                 except (ValueError, IndexError):
                     pass
                 else:
+                    full_edge.add((s, t))
                     if border is None:
                         fill = _color_diff(p, background) <= thresh
                     else:
                         fill = p != value and p != border
                     if fill:
                         pixel[s, t] = value
                         new_edge.add((s, t))
-                        full_edge.add((s, t))
-        full_edge = edge  # do not record useless pixels to reduce memory consumption
+        full_edge = edge  # discard pixels proceeded
```

Comment:
Typo? processed, not proceeded

Reply:
🤦‍♂️ just fixed it. blindly copy-paste from my own comment without double check...

Reply:
hmmm... strange, editted comment led to Build execution time exceeded. Maybe I should add a tag in commit message to skip the build test. What's tag should I use?

Reply:
Thanks. In case you're interested for the future, to skip the tests, include ' [ci skip]' in the commit message.